cg_hermes/config/balsamic.py

```diff
@@ -1,601 +1,738 @@
 """Tags that are defined in Balsamic deliverables mapped to tags used in CG
 
-The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not mandatory by default.
-However the tags that are available to a particular analysis is mandatory for that analysis.
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However the tags that are available to a particular analysis is mandatory for that analysis.
 """
 
+TAGS = {
+    # QC & reports
+    "multiqc_report.html": ["multiqc-html", "html"],
+    "multiqc_data.json": ["json", "multiqc-json"],
+    "report.html": ["balsamic-report"],
+    "config.json": ["balsamic-config"],
+    "BALSAMIC_X.X.X_graph.pdf": ["balsamic-dag"],
+    "metrics_deliverables.yaml": ["qc-metrics-yaml", "yaml"],
+    "concatenated_*_R_fastp.html": [
+        "quality-trimmed-fastq-html",
+        "html",
+    ],
+    "concatenated_*_R_fastp.json": [
+        "json",
+        "quality-trimmed-fastq-json",
+    ],
+    # Raw data & alignment files
+    "concatenated_*_R_1.fp.fastq.gz": [
+        "quality-trimmed-fastq-read1",
+        "read1",
+    ],
+    "concatenated_*_R_2.fp.fastq.gz": [
+        "quality-trimmed-fastq-read2",
+        "read2",
+    ],
+    "tumor.merged.cram": ["tumor-cram", "cram"],
+    "tumor.merged.cram.crai": ["tumor-cram-index", "cram"],
+    "normal.merged.cram": ["normal-cram", "cram"],
+    "normal.merged.cram.crai": ["normal-cram-index", "cram"],
+    # tnscope (WGS) & vardict (PANEL)
+    "tnscope.all.vcf.gz": ["tnscope", "vcf-all", "snv", "annotated-somatic-vcf-all"],
+    "tnscope.all.vcf.gz.tbi": ["tnscope", "annotated-somatic-vcf-all-index", "vcf-all", "snv"],
```

Comment:
```suggestion
    "tnscope.all.vcf.gz": ["tnscope", "vcf-all", "snv", "annotated-somatic-vcf-all"],
    "tnscope.all.vcf.gz.tbi": ["tnscope", "annotated-somatic-vcf-all-index", "vcf-all", "snv"],
```


Regarding the order of tags in the list ["tnscope", "vcf-all", "snv", "annotated-somatic-vcf-all"], can it be of any order? I see for some tags  it follows other random order `["annotated-somatic-vcf-all", "snv", "vardict", "vcf-all"]`. Does it matter with the order ?

Reply:
The order doesn't matter, for most of these files I just copied the tags as they appear in the `.hk` file. I'll try to standardise the order.